index.js

```diff
@@ -28,6 +28,9 @@ module.exports = {
     getStore: true,
     getStoreWithEntities: true,
     getCurrentRoute: true,
+    ReduxState: 'readonly',
```

Comment:
I'd also recommend moving these to the molly repo, since `ReduxState` is specific to our project. Even `React`.